src/AuthModule/Components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,7 +1,146 @@
-import React from 'react'
+import { Button, Dialog, DialogContent,  Stack, TextField, Typography } from '@mui/material'
+import axios from 'axios';
+import React, { useState } from 'react'
+import { useForm } from 'react-hook-form';
+import { baseUrl } from '../../../Constants/Components/Urls';
+import RemoveRedEyeIcon from '@mui/icons-material/RemoveRedEye';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import VisibilityOffIcon from '@mui/icons-material/VisibilityOff';
+
+export default function ChangePassword({handleClose}) {
+  
+    const [open, setOpen] = useState(false);
+    
+    const [showPasswordOldPassword, setShowPasswordOldPassword] = useState(true);
+  const [showPasswordNewPassword, setShowPasswordNewPassword] = useState(true);
+  const [showPasswordConfirmPassword, setShowPasswordConfirmPassword] =useState(true);
+
+  const togglePasswordVisibilityOldPassword = () => {
+    setShowPasswordOldPassword(!showPasswordOldPassword);
+  };
+
+  const togglePasswordVisibilityNewPassword = () => {
+    setShowPasswordNewPassword(!showPasswordNewPassword);
+  };
+
+  const togglePasswordVisibilityConfirmPassword = () => {
+    setShowPasswordConfirmPassword(!showPasswordConfirmPassword);
+  };
+   
+
+    const {register,handleSubmit,watch, formState: { errors },setValue}=useForm();
+
+const submitData=async(data:IChangePassword)=>{
+try {
+  const response=await axios.post(`${baseUrl}/admin/users/change-password`,data,
+  {headers:{Authorization:"Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI2NjFhYTZkNzZlYmJiZWZiYzE5ZjMyM2UiLCJyb2xlIjoiYWRtaW4iLCJ2ZXJpZmllZCI6ZmFsc2UsImlhdCI6MTcxMzAyNDQ1MCwiZXhwIjoxNzE0MjM0MDUwfQ.WGP2BzDpvgwhKVCenvoDk7yOr4GnG4utAhSnf-baTOg"}})
+  console.log(response)
+} catch (error) {
+  
+}
+}
+
 
-export default function ChangePassword() {
   return (
-    <div>ChangePassword</div>
+    <div>
+      <div className='text-center'>
+        <h2>Change Password</h2>
+      </div>
+         
+          <form  onSubmit={handleSubmit(submitData)}>
+      <Stack spacing={2} margin={2}>
+
+      <div className='d-flex form-outline  position-relative'>
+   <TextField
+   sx={{width:"100%"}}
+   type={!showPasswordOldPassword ? "text" : "password"}
+   {...register("oldPassword")}
+       variant="filled"
+       size="small"
+        label="old Password"
+        name="oldPassword"
+        required
+      />
+                <ListItemIcon
```

Comment:
It is also common when using iconButton to have a tooltip component that describes what this button do like this one from YouTube home page:
![image](https://github.com/AnasKhaled7/hotel-system-react/assets/64327685/cb8ea408-ab91-47e5-b786-3ee9ee6cb57d)
This will help users unfamiliar with the icon to know what this button do.
You can look [here](https://mui.com/material-ui/react-tooltip/) for MUI tooltip component